index.html

```diff
@@ -0,0 +1,207 @@
+<!doctype html>
+<html>
+<head>
+<title>Nicholas Botorog</title>
+<meta charset="UTF-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<link rel="stylesheet" href="style.css" >
+<script src="https://kit.fontawesome.com/00804bacca.js" crossorigin="annonymous"></script>
+</head>
+<body>
+    <nav class="navbar">
+        <ul class="nav-links">
+          <li class="nav-item"><a href="#home">Home.</a></li>
+          <li class="nav-item"><a href="#about">About.</a></li>
+          <li class="nav-item"><a href="#experience">Experience.</a></li>
+          <li class="nav-item"><a href="#projects">Projects.</a></li>
+        </ul>
+      </nav>
+    
```

Comment:
Some really comprehensive use of specific, semantic html elements.  Also, your attribute naming is clear, effective, and intuitive which makes your code easy to follow.